### ISSUE TITLE
Cant access Remote Configuration Tool. & Speech Recognition Issue (Remote is listening)

### ISSUE THREAD

**Issue by Djnightfires** [evancohen/smart-mirror#685]
Hello

Two issue's I've come across if anyone wants to help/guide me out of this issues.

First issue is the Remote Configuration Tool.
I was able to access the Remote Configuration Tool the first (and few more times in that session) but can no longer access the Remote Configuration Tool despite putting in the right link. For me it was http://192.168.43.75:8080/config.html and I've tried using different connections, different computers using with both ssh & without ssh (and a phone which worked the first session) but couldn't find a solution to it. I checked the file using WINSCP on a computer (ssh) just to see if I could find anything suspicious but nothing out of the ordinary.

Second issue is the Speech Recognition. 
I followed the guide best I could & got to the point where starting up the smart mirror it says Remote listening on http://192.168.43.75:8080 which it used to display a wall of errors but no longer does that. I tried running sonus using npm run sonus but all it did was 

> smart-mirror@0.0.14 sonus /home/pi/smart-mirror
> node sonus.js

~/smart-mirror $ > npm run sonus

It didn't show any blinking cursor like the guide said. It just went back to what I entered in

Supposedly-
The installed all necessary dependencies is done properly
The microphone configured correctly (I'm just a lil cautious at the moment when starting the mirror if its using the right device)
The Google Speech Platform is I assumed sorted (followed the steps, but the file in the smart mirror directory, when starting the mirror I use to have errors but errors were gone & its showing the its remote listening) but the sonus not listening/no blinking cursor when using npm run sonus has me question otherwise.

This is what happens when I start up the mirror with or without ssh

:~/smart-mirror $ DISPLAY=:0 npm start

> smart-mirror@0.0.14 start /home/pi/smart-mirror
> electron main.js

Remote listening on http://192.168.43.75:8080

Any help would be appreciated as I would really love to get this working.

**Comment by Djnightfires** [comment]
Nevermind I sorted out the first issue (I was typing in a different http address to what I had)

**Comment by Djnightfires** [comment]
Progressed further since I had access to the remote access on a phone, changed recording device to my usb mic so now it can pick up stuff, changed google cloud speech keyfile from ./keyfile.json to keyfile.json (not sure if that makes a difference since I called I just named mine in the smart-mirror folder keyfile.json. 

Tried npm run sonus, It now gives me a cursor (does'nt blink but still works) I say smart mirror hotword and sometimes it picks it up sometimes its not (could just be a poorly done voice recording but it works) no partial commands happen at all no matter what I say and some final commands happen every known and again and one streaming error saying (refering to the image I pasted in)

![20170927_134159](https://user-images.githubusercontent.com/32131044/30914099-4ec25dc0-a38a-11e7-8882-cf6330436c6e.jpg)

and I'm pretty much stuck here at this point.

**Comment by evancohen** [comment]
Well, that's a new error! That error looks like something is up with gRPC, but since you aren't getting partial results I'm wondering is something else might be the problem...

A bit of a shot in the dark, but you could try upgrading to the beta release of Sonus to see if that fixes this issue for you. From the `smart-mirror` directory, run:
```
npm install sonus@next
```

Give that a shot and let me know if it works for ya.

**Comment by Djnightfires** [comment]
Tried npm install sonus@next and not sure if it did anything to be honest. I tried running npm run sonus to test the voice results & weird enough everything I said was h1. No partials,finals or errors just h1's.

I tried messing about with the keyfile, ssh, changing connections, changing devices & few other stuff I cant remember honestly as it was more or less of trying everything until something changed & that somehow fixed the sonus somehow,

I can now start up the smart-mirror and successfully tell smart mirror to example giphy dog & it picks it up & displays a gif of a dog no problem.

My last issue is how do I activate the rest of the modules to work such as last.fm, rss feed, as the 'What can I say command' isn't showing me the commands for those modules/its not starting up automatically (Sorry if missed something obvious I'm rather new to the Raspberry pi). Also show me command isn't working but not sure if its my youtube API key or something different.

Thanks for the help
